aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -97,17 +97,17 @@ app.UseFacebookAuthentication(new FacebookOptions()
 
 Run your application and click **Log in**. You will see an option to sign in with Facebook.
 
-![image](index/_static/DoneFacebook.png)
+![Web application: User not authenticated](index/_static/DoneFacebook.png)
 
 When you click on Facebook, you will be redirected to Facebook for authentication.
 
-![image](index/_static/FBLogin2.png)
+![Facebook authentication page](index/_static/FBLogin2.PNG)
```

Comment:
Rather than changing extension to upper case in .md, would be preferable to change the file extension to lower case.  That would require name change of the file, as git won't register case-only change as a change.  So e.g., change this to FBLogin2a.png and change the filename to the same value.  There are two more .PNG files in this folder, pass1.PNG and pass2.PNG